### PR TITLE
Bump GNOME runtime version to 47

### PR DESCRIPTION
**Pull request by acerspyro**

### CODE REVIEW

File: data/io.github.flattool.Warehouse.metainfo.xml.in [hunk 1]
```diff
@@ -64,6 +64,14 @@
     </screenshot>
   </screenshots>
   <releases>
+      <release version="1.6.5" date="2024-9-18" timestamp="1720282600">
```
Comment (acerspyro) [inline data/io.github.flattool.Warehouse.metainfo.xml.in:67]: Not sure if you were using a tool to manage the releases, but I haven't changed the timestamp from the previous release, just the date.

File: .editorconfig
Comment (acerspyro) [inline .editorconfig:1]: I added a .editorconfig to the project as my VSCode really wanted to format JSON files with two space indentation.

**Comment by heliguy4599** [comment]
Thank you! This was handy. I'll merge it today, and release tomorrow.

**Comment by heliguy4599** [comment]
Tested it now, everything is in shape. Once again thank you!

**Pull request merged**